sys-utils/nsenter.c

```diff
@@ -202,6 +207,50 @@ static int get_ns_ino(const char *path, ino_t *ino)
 	return 0;
 }
 
+static void open_cgroup_procs(void)
+{
+	char *buf = NULL, *path = NULL;
+	int cgroup_fd;
+	char fdpath[PATH_MAX];
+
+	open_target_fd(&cgroup_fd, "cgroup", optarg);
+
+	if (read_all_alloc(cgroup_fd, &buf) < 1)
+		err(EXIT_FAILURE, _("failed to get cgroup path"));
+
```

Comment:
There should be also free() somewhere ;-) Fixed by https://github.com/util-linux/util-linux/commit/c67a2b842d7a6479114caf8f90965c045d491ac2